src/map/include/items/inventory.h

```diff
@@ -18,10 +19,13 @@ enum class ReturnValue {
 
 // returns the first available spot in the inventory or 0 if there is no more space
 size_t get_first_available_spot(const EntitySystem& entitySystem, RoseCommon::Entity entity, RoseCommon::Entity item = entt::null);
+uint8_t has_bullets(const EntitySystem& entitySystem, RoseCommon::Entity entity);
```

Comment:
Not sure if this is a good name for what the function does. As far as I understand this function returns the slot of the used bullets, or 0 otherwise. Maybe something like `get_bullet_slot` ? And it would return an std::option<uint8_t>, with no value meaning no bullet slot.

Tbh I think we should avoid exposing inventory slots as much as possible outside of inventory management, and deal with entities directly. This would allow us to change how the inventory is managed in the future if we require, without having to chase everywhere in the code.